purchases-js-hybrid-mappings/src/index.ts

```diff
@@ -0,0 +1 @@
+export { PurchasesCommon } from './purchases-common';
```

Comment:
We only need to export `PurchasesCommon` at least for now. We won't be exposing other types from the library, since everything will be basic types.